src/screens/balanceWithdrawal/ConfirmWithdrawPix.tsx

```diff
@@ -0,0 +1,244 @@
+                flexDirection: "row",
+                alignItems: "center",
+                gap: spacing[4]
+              }}
+            >
+              <View style={styles.imageWrapper}>
+                <Image source={selectedCurrency.image} style={styles.image} />
+              </View>
+
+              <Label style={{ marginTop: 2 }}>
+                {selectedCurrency.currency}
+              </Label>
+            </View>
+
+            <View style={{ alignItems: "flex-end", gap: spacing[4] }}>
+              <Heading size="heading2">
+                {Number(selectedCurrency.amount).toLocaleString(
+                  "pt-BR",
+                  SIMPLECURRENCYFORMAT
+                )}
+              </Heading>
+
+              <Heading size="heading5" style={{ color: dark.fg.muted }}>
+                {Number(selectedCurrency.amount).toLocaleString(
+                  "pt-BR",
+                  CURRENCYFORMAT
+                )}
+              </Heading>
+            </View>
+          </View>
+        </View>
+
+        <Divider />
+
+        <View style={styles.listItem}>
+          <Label size="md-regular" style={styles.labelGrey}>
+            Taxas
+          </Label>
+
+          <Label>0,5% Transfero</Label>
+        </View>
+
+        <View style={styles.listItem}>
+          <Label size="md-regular" style={styles.labelGrey}>
+            Chave pix
+          </Label>
+
+          <Label>{pixKey}</Label>
+        </View>
+
+        <View style={styles.listItem}>
+          <Label size="md-regular" style={styles.labelGrey}>
+            Data
+          </Label>
+
+          <Label>
+            {date.toLocaleDateString("pt-BR", {
+              day: "numeric",
+              month: "short",
+              year: "numeric"
+            })}
+          </Label>
+        </View>
+
+        <View style={styles.listItem}>
+          <Label size="md-regular" style={styles.labelGrey}>
+            Horário
+          </Label>
+
+          <Label>
+            {date.toLocaleTimeString("pt-BR", {
+              hour: "numeric",
+              minute: "numeric"
+            })}
+          </Label>
+        </View>
+
+        <View style={styles.listItem}>
+          <Label size="md-regular" style={styles.labelGrey}>
+            ID da transação
+          </Label>
+
+          <Label>T</Label>
+        </View>
+      </View>
+
+      <View style={{ marginTop: "auto" }}>
+        <Button
+          text="Confirmar pedido de saque"
+          onPress={() => {
+            handlePresentModalPress();
+
+            onConfirm();
+          }}
+        />
+      </View>
+
+      <BottomSheet bottomSheetModalRef={bottomSheetModalRef} height="90%">
+        {isLoading ? (
+          <View
+            style={{
+              flex: 1,
+              alignItems: "center",
+              justifyContent: "center",
+              gap: spacing[8]
+            }}
+          >
+            <Label size="md-bold">Sending Transaction...</Label>
```

Comment:
It should be in portuguese

```suggestion
            <Label size="md-bold">Enviando Transação...</Label>
```